app/build.gradle

```diff
@@ -0,0 +1,75 @@
+apply plugin: 'com.android.application'
+
+def file = rootProject.file("local.properties");
+def prop = new Properties()
+prop.load(new FileInputStream(file))
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        applicationId "com.ybsystem.tweethub"
+        minSdkVersion 26
+        targetSdkVersion 29
+        versionCode 13
+        versionName "1.5.0"
+        buildConfigField("String", "CK", prop['ck'])
+        buildConfigField("String", "CS", prop['cs'])
+        buildConfigField("String", "CB", prop['cb'])
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+            shrinkResources true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            minifyEnabled true
+            shrinkResources true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    lintOptions {
+        disable 'SpUsage'
+        disable 'SetTextI18n'
+        disable 'IconLocation'
+        disable 'HardcodedText'
+        disable 'UnusedResources'
+        disable 'ContentDescription'
+        disable 'NonConstantResourceId'
+        disable 'AnnotationProcessorOnCompilePath'
+    }
+}
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ybs32/TweetHub-for-Android/blob/795063919f5dbe16c86520b8361d5d49a353b825/app/build.gradle#L51"></span>A newer version of com.google.android.material:material than 1.2.1 is available: 1.3.0</td>
    </tr>
  </tbody>
</table>